components/inputs/input-date.js

```diff
@@ -33,6 +33,10 @@ class InputDate extends FormElementMixin(LocalizeCoreElement(LitElement)) {
 			 * Text to reassure users that they can choose not to provide a value in this field (usually not necessary)
 			 */
 			emptyText: { type: String, attribute: 'empty-text'},
+			/**
+			 * Style the component as invalid
+			 */
+			invalid: { type: Boolean, reflect: true },
```

Comment:
I'm late to the party here, so this may have been discussed already -- I think Allan's already adding this to attribute to `FormElementMixin`?